articles/install-guide/csinstall.md

```diff
@@ -10,7 +10,9 @@ uid: microsoft.quantum.install.cs
 # Develop with Q# and .NET
 
 Q# is built to play well with .NET languages such as C# and F#.
-In this guide, we'll demonstrate how to use Q# with a host program written in a .NET language.
+In this guide, we demonstrate how to use Q# with a host program written in a .NET language.
+
+First we create the Q# library and .NET host, and then demonstrate how to call to Q# from the host.
```

Comment:
Strictly speaking, the Q# project defines a quantum executable as entry points aren't allowed in libraries.
```suggestion
First we create the Q# application and .NET host, and then demonstrate how to call to Q# from the host.
```